.github/workflows/terraform.yml

```diff
@@ -12,5 +12,10 @@ on:
       - "**"
 
 jobs:
-  terraform_format_test:
-    uses: ./.github/workflows/terraformformat.yml
+  terraform-format-test:
+    uses: ./.github/workflows/terraform_format.yml
+  tfsec-pr-commenter:
+    uses: ./.github/workflows/tfsec_pr_commenter.yml
+  terraform-docs:
+    needs: [terraform-format-test, tfsec-pr-commenter]
```

Comment:
We could probably change it to only run on PR's, the tfsec one just won't do anything on push. I had it set to all pushes because I was pushing directly to master on some of the initial commits but I'll change it to only PR's since that makes more sense.